nilearn/plotting/img_plotting.py

```diff
@@ -366,34 +366,39 @@ def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
         black_bg: boolean, optional
             If True, the background of the image is set to be black. If
             you wish to save figures with a black background, you
-            will need to pass "facecolor='k', edgecolor='k'" to pylab's
-            savefig.
+            will need to pass "facecolor='k', edgecolor='k'" to plt.savefig.
         cmap: matplotlib colormap, optional
             The colormap for the anat
+        vmin: float
+            Lower bound for plotting, passed to plt.imshow
+        vmax: float
+            Upper bound for plotting, passed to plt.imshow
 
         Notes
         -----
         Arrays should be passed in numpy convention: (x, y, z)
         ordered.
     """
-    anat_img, black_bg, vmin, vmax = _load_anat(anat_img,
-                                                dim=dim, black_bg=black_bg)
-    # vmin and/or vmax could have been provided in the kwargs
-    vmin = kwargs.pop('vmin', vmin)
-    vmax = kwargs.pop('vmax', vmax)
+    anat_img, black_bg, anat_vmin, anat_vmax = _load_anat(
+        anat_img,
+        dim=dim, black_bg=black_bg)
+
+    vmin = vmin or anat_vmin
+    vmax = vmax or anat_vmax
```

Comment:
Isn't that really fragile: if I pass vmin=0, it will use anat_vmin. I think that I would prefer something more explicit.